jbpm-console-ng-business-domain/jbpm-console-ng-business-domain-backend/src/test/java/org/jbpm/console/ng/bd/integration/KieServerDataSetProviderTest.java

```diff
@@ -84,4 +119,40 @@ public void appendBetweenIntervalSelectionTest() {
         assertEquals(Double.valueOf(maxValue),filterParams.get(0).getValue().get(1));
     }
 
+    @Test
+    public void lookupDataSetLogicalExprTest(){
+        DataSetLookup lookup = new DataSetLookup();
+        lookup.setDataSetUUID("");
+        when(dataSetDef.getUUID()).thenReturn("");
+
+        final ColumnFilter testFilter = OR( likeTo("column1","%value%"),likeTo("column2","%value%"));
+
+        if (lookup.getFirstFilterOp() != null) {
+            lookup.getFirstFilterOp().addFilterColumn(testFilter);
+        } else {
+            DataSetFilter filter = new DataSetFilter();
+            filter.addFilterColumn(testFilter);
+            lookup.addOperation(filter);
+        }
+        try {
+            kieServerDataSetProvider.lookupDataSet(dataSetDef ,ConsoleDataSetLookup.fromInstance(lookup,"servereTemplateId"));
+
+            final ArgumentCaptor<QueryFilterSpec> captorEdit = ArgumentCaptor.forClass(QueryFilterSpec.class);
+            verify(queryServicesClient).query( anyString(),anyString(), captorEdit.capture(), anyInt(),anyInt(), any());
+
+            assertNotNull(captorEdit.getValue());
+            QueryParam[] parameters =captorEdit.getValue().getParameters();
+            assertEquals(1,parameters.length);
+
+            List<CoreFunctionFilter> expr = (List<CoreFunctionFilter>)parameters[0].getValue();
+            assertEquals("OR",parameters[0].getOperator());
+
+            assertEquals("column1 like %value%, true",expr.get(0).toString());
+            assertEquals("column2 like %value%, true",expr.get(1).toString());
+
+        }catch (Exception e){
+            fail();
```

Comment:
I believe this is not needed. You can add `throws Exception` into the method signature. This is fine in case of test methods and simplifies the test code a bit.